package.json

```diff
@@ -28,7 +28,9 @@
     "@antora/site-mapper": "3.0.0-alpha.1",
     "@antora/site-publisher": "3.0.0-alpha.1",
     "@antora/ui-loader": "3.0.0-alpha.1",
-    "cheerio": "^1.0.0-rc.3"
+    "asciidoctor-jupyter": "git+https://github.com/neo4j-documentation/asciidoctor-jupyter.git",
```

Comment:
We should release/tag https://github.com/neo4j-documentation/asciidoctor-jupyter before merging